### DISCUSSION TITLE
cannot play videos when routed to virtual speaker (since after v21.8.0)

### DISCUSSION THREAD

**Discussion by pwaser** [smplayer-dev/smplayer#626] [General]
I've been using smplayer for a long time without any problem. But I recently did a clean Windows install, and I noticed that when I set (route) SMPlayer to play on one of my virtual speakers (from my audio interface RME Babyface Pro FS) with a Windows store app called "Ear Trumpet", the player no longer works for video, the play button doesn't play the video; but with audio files it works normally, and plays on the virtual speaker. I've been playing videos with audio routed to the virtual speaker for 1 year, and now it suddenly doesn't work. So after many experiments, I started  testing older versions, and voilá, version **v21.8.0** and older work flawlessly with the audio rerouted to a virtual speaker that is not the default speaker (even though said default speaker is also a virtual speaker), so the problem had come from downloading the latest version when performing my clean Windows installation.
Can you please identify the issue with the latest versions? I need to lay this on a virtual speaker for streaming purposes, so I can mute it in OBS. Otherwise, I'm stuck with this version for eternity.

Here's a video where I demonstrate the issue by installing 2 versions of the programme and trying to play a video after rerouting to a to virtual speaker different from the default device:
[https://www.youtube.com/watch?v=a7E9gTVXO5Q&feature=youtu.be&ab_channel=WadokinCloudwalker](url)

**Comment by freekeee** [comment]
i have the same experience - i am using VoiceMeeter virtual input/outputs.  As soon as I change the physical output - video begins working.

**Reply by pwaser** [reply]
I see I'm not the only one using virtual speakers with this programme. Does video reproduction work without problems when you use version 21.8.0? I downloaded it from fosshub, pressing the "older versions" button.
[https://www.fosshub.com/SMPlayer-old.html](url)
